pkg/cmd/admin/registry/registry.go

```diff
@@ -299,6 +302,9 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	env := app.Environment{}
 	env.Add(secretEnv)
 
+	if cfg.EnforceQuota {
```

Comment:
Why do we need this flag in this command?  Why would we ever want this disabled, or give admins the choice?